docs/reuse/cis-note.md

```diff
@@ -0,0 +1,3 @@
+:::tip
+For apps to perform security monitoring of your cloud infrastructure, see [Cloud Infrastructure Security](/docs/cloud-infrastructure-security/).
```

Comment:
The way this is written is a little unclear. I recommend either: 

_To view apps to perform security monitoring of your cloud infrastructure..._ 
or 
_To enable apps to perform security monitoring of your cloud infrastructure..._

Reply:
I like the second choice.